src/MassTransit.RabbitMqTransport/Transport/RabbitMqSendTransport.cs

```diff
@@ -144,6 +145,7 @@ public async Task Send(ModelContext modelContext)
                 }
                 catch (Exception ex)
                 {
+                    Console.WriteLine("FUCK: {0}", ex.Message);

```

Comment:
Will it stay here or just a left over @phatboyg ;)?

Reply:
LOL that commit never made it into the final branch. :)

Reply:
Haha, ok, just wanted to be sure :)